test/unit/views/ConsignmentReferenceViewSpec.scala

```diff
@@ -37,6 +37,8 @@ class ConsignmentReferenceViewSpec extends ViewSpec with MockitoSugar with Befor
 
   private val page = injector.instanceOf[consignment_references]
 
+  private val goodsDirection = JourneyType.ARRIVE
```

Comment:
Nit: Similar question about the name of this field. Is it correct?

Reply:
The two journey types relevant to the ConsignmentReferences screen are ARRIVE & DEPART so I though it was appropriate to say they are directions of travel for the goods. Happy to accept better suggestions though :)